Example/Sentinel/AppDelegate.swift

```diff
@@ -67,31 +66,25 @@ private extension AppDelegate {
         )
     }
 
-    var optionSwitchItems: [ToolTableSection] {
+    var optionSwitchItems: [PreferencesTool.Section] {
         [
             .init(
```

Comment:
For better readability use class names instead of `.init`. Also, update that through the whole codebase if applicable.

We are trying to make examples and our code as readable as possible.